presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -115,6 +115,7 @@
 
```

Comment:
Please add "end-to-end" test where you SELECT unsupported type as varchar. Maybe to `io.prestosql.plugin.jdbc.TestJdbcIntegrationSmokeTest` or somewhere close.

Reply:
A test in `TestPostgreSqlTypeMapping` would be nice too.